scripts/konlet-startup

```diff
@@ -28,6 +28,28 @@ if [[ $? -ne 0 ]]; then
   ACTUAL_KONLET_VERSION="${KONLET_VERSION_TAG}"
 fi
 
+sudo sh -c 'echo "[Unit]
+Description=A daemon to collect Docker events and send to to journal logs.
+After=docker.socket
+Wants=docker.socket
+
+[Service]
+ExecStart=/usr/bin/docker events
+Restart=on-failure
+RestartSec=5
+" >> /etc/systemd/system/stackdriver-docker-events-collector.service'
+
+sudo sh -c 'echo "# Collects logs generated from Docker events.
+<source>
+  @type systemd
+  filters [{ \"_SYSTEMD_UNIT\": \"stackdriver-docker-events-collector.service\" }]
+  pos_file /var/log/google-fluentd/stackdriver-docker-events.log.pos
+  read_from_head true
+  tag cos_docker_events
+</source>" >> /etc/stackdriver/logging.config.d/fluentd-lakitu.conf'
+
+sudo systemctl start stackdriver-docker-events-collector.service
+sudo systemctl restart stackdriver-logging.service
```

Comment:
Are you positive that systemctl ran from inside of a systemctl-launched script won't cause any deadlock issues?